### ISSUE TITLE
App crash

### ISSUE THREAD

**Issue by mfbx9da4** [bamlab/react-native-image-resizer#379]
We have sometimes seen the following crashes in production

```
Crashed: com.apple.root.default-qos
0  libdispatch.dylib              0x2984 _os_object_release + 76
1  Xxxx                           0x50ebe4 -[RCTImageLoader imageURLLoaderForURL:] + 184 (RCTImageLoader.mm:184)
2  Xxxx                           0x50fa80 -[RCTImageLoader _loadImageOrDataWithURLRequest:size:scale:resizeMode:priority:attribution:progressBlock:partialLoadBlock:completionBlock:] + 497 (RCTImageLoader.mm:497)
3  Xxxx                           0x511f90 -[RCTImageLoader loadImageWithURLRequest:size:scale:clipped:resizeMode:priority:attribution:progressBlock:partialLoadBlock:completionBlock:] + 834 (RCTImageLoader.mm:834)
4  Xxxx                           0x50f328 -[RCTImageLoader loadImageWithURLRequest:size:scale:clipped:resizeMode:priority:progressBlock:partialLoadBlock:completionBlock:] + 382 (RCTImageLoader.mm:382)
5  Xxxx                           0x50f220 -[RCTImageLoader loadImageWithURLRequest:size:scale:clipped:resizeMode:progressBlock:partialLoadBlock:completionBlock:] + 361 (RCTImageLoader.mm:361)
6  Xxxx                           0x93f09c __109-[ImageResizer createResizedImage:width:height:format:quality:rotation:outputPath:keepMeta:options:callback:]_block_invoke + 399 (RCTImageResizer.m:399)
7  libdispatch.dylib              0x2320 _dispatch_call_block_and_release + 32
8  libdispatch.dylib              0x3eac _dispatch_client_callout + 20
9  libdispatch.dylib              0x6f8c _dispatch_queue_override_invoke + 788
10 libdispatch.dylib              0x15944 _dispatch_root_queue_drain + 396
11 libdispatch.dylib              0x16158 _dispatch_worker_thread2 + 164
12 libsystem_pthread.dylib        0xda0 _pthread_wqthread + 228
13 libsystem_pthread.dylib        0xb7c start_wqthread + 8
```

What could cause this issue?

**Comment by mfbx9da4** [comment]
That won't be possible, close for now

**Comment by taboulot** [comment]
OK,

FYI, just did a small example on [this repository](https://github.com/taboulot/TestImageResizer/tree/rn-image-resizer-3.0.5) with everything working. Hope it can help.

We know that we also have some issues with iOS. When this PR will be merged #381, a new version will be released.

Until then, I close this issue & let you reopen it if you need it.
